source/databricks/tests/integration/calculator/test_calculator.py

```diff
@@ -124,6 +157,11 @@ def test_calculator_job_creates_files_for_each_gridarea(
         "2022-09-01T22:00:00Z",
     ]
 
+    # Reads time_series_points from parquet into dataframe
+    actual_time_series_points = spark.read.parquet(
```

Comment:
I normally read "actual" as the result of my action/sut. How about this?

```suggestion
    input_time_series_points = spark.read.parquet(
```

Reply:
done